Sound/Tidal/Pattern.hs

```diff
@@ -317,16 +319,16 @@ playWhen test (Pattern f) = Pattern $ (filter (\e -> test (eventOnset e))) . f
 playFor :: Time -> Time -> Pattern a -> Pattern a
 playFor s e = playWhen (\t -> and [t >= s, t < e])
 
-{- | There is a similar function named `seqP` which allows you to define when
+{- | The function @seqP@ allows you to define when
 a sound within a list starts and ends. The code below contains three
-separate patterns in a "stack", but each has different start times 
```

Comment:
I think, since `seqP` really uses `stack` we can make it a linked reference to `stack`.